cli/src/main/scala/aqua/AppOpts.scala

```diff
@@ -193,6 +193,12 @@ object AppOpts {
       .map(_ => true)
       .withDefault(false)
 
+  val scriptOpt: Opts[Boolean] =
+    Opts
+      .flag("script", "Generate code for script storage. Will ignore other options")
```

Comment:
I think `--script` is kinda confusing. Everything you write in Aqua is a script.

I think `--scheduled` or maybe `--background` would be better.

```
.flag("scheduled", "To use as a scheduled script. Same as --air --no-relay --no-xor.")
```